pagopa/io-app
# feat: [PE-615] Change labels in CGN card detail screen

## Short description
Change labels in CGN card detail screen

## List of changes proposed in this pull request
- change labels in localization files

## How to test
Related figma https://www.figma.com/design/yOoBGDr1MJpa0LY5lDUFSd/IOapp_CartaGiovaniNazionale?node-id=6171-28555&t=gaDY2KPmmBJvThH2-4
Navigate to highlighted screens to check new copy.
- login into the app -> navigate to "portafoglio" screen clicking on the icon in the bottom menu -> navigate to "carta giovani nazionale" by clicking on the the card in the wallet -> check copy on this screen 
- login into the app -> navigate to "portafoglio" screen clicking on the icon in the bottom menu -> navigate to "carta giovani nazionale" by clicking on the the card in the wallet -> click on "i" info icon button nearby "carta eyca" -> check copy on this screen 
- login into the app -> navigate to "portafoglio" screen clicking on the icon in the bottom menu -> navigate to "carta giovani nazionale" by clicking on the the card in the wallet -> click on "deactivate carta giovani nazionale button" -> check copy on this screen 

@mariateresaventura 

[PE-615]: https://pagopa.atlassian.net/browse/PE-615?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
